File: C_Fundamentals/Command.c
#include<stdio.h>
#include<stdlib.h>

int main(int argc ,char *argv[])
{
    int ans = 0;
    printf("Name of executable is : %s\n",argv[0]);

    printf("number of command line arguments are : %d\n",argc);

    ans = atoi(argv[1]) + atoi(argv[2]);
    
    printf("Addition is : %d\n",ans);
    
    return 0;
}

File: Java_Fundamentals/Command.java
import java.lang.*;


class Command
{
    public static void main(String Arr[])
    {   
        System.out.println("Number of command line argument is  : "+ Arr.length);
        System.out.println("First Number is : "+ Arr[0]);
        System.out.println("First Number is : "+ Arr[1]);

        int ans = Integer.parseInt(Arr[0])+ Integer.parseInt(Arr[1]);
        System.out.println("Addition is : "+ ans);

    }
} 